src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -541,6 +541,19 @@ public ResponseEntity<GroupingDescription> groupingDescription(
                 .body(groupingOwnerService.groupingsDescription(currentUser, path));
     }
 
+    /**
+     * Get a list of members for a selected grouping path with search string.
+     */
+    @GetMapping(value = "/groupings/{path:[\\w-:.]+}/{searchString}")
```

Comment:
I know I put this on the ticket for the URL, but now I'm looking at it, it is more RESTful if the URL was:

/groupings/{path}/search/{search}

Reply:
Sounds good for the new search grouper functions in the future